packages/mongo-nav/src/types.ts

```diff
@@ -1,3 +1,14 @@
+const OnElementClick = {
```

Comment:
Just to spice things up...I have half a mind that this should legitimately be an enum, such that we completely control the cases and force end users to use the enum to switch behaviors off of.

I might be getting ahead of myself tho, so this is still fine as is.